packages/schemer/src/index.ts

```diff
@@ -148,7 +149,10 @@ export default class Schemer {
 
   async _validateAssetsAsync(data: any) {
     const assets: AssetField[] = [];
-    traverse(this.schema, { allKeys: true }, (subSchema, jsonPointer, a, b, c, d, property) => {
+
+    const schema = await JsonSchemaRefParser.dereference(this.schema);
```

Comment:
Dereference the schema in the constructor so that this issue is fixed for all fields